bin/gsl.py

```diff
@@ -6,21 +6,28 @@
 import os
 import hashlib
 import logging
-from gsl.utils import yield_packages, package_name, PACKAGE_SERVER, depot_url
+import gsllib
+from gsllib.utils import yield_packages, package_name, PACKAGE_SERVER, depot_url
```

Comment:
if we're renaming, what do you think of renaming the package to something cargo-port related?